lib/db/StudyRequestBulkDAO.js

```diff
@@ -97,6 +98,36 @@ INSERT INTO "study_requests_bulk" (
     return StudyRequestBulk.read.validateAsync(studyRequestBulk);
   }
 
+  static async byLocationsSelection(locationsSelection) {
```

Comment:
rename, and actually handle pending.  a bulk study request is pending if it has at least one non-closed study request.